### PR TITLE
Repository routes & tests

### PR DESCRIPTION
**Pull request by samilaaroussi**

### CODE REVIEW

**Comment by rjollet** [comment]
Did you do the spec ?

**Comment by samilaaroussi** [comment]
@rjollet No, I forgot it. I'm working on it now.

File: spec/repo_spec.rb [hunk 1]
```diff
@@ -14,20 +14,20 @@
     VCR.eject_cassette
   end
 
-  describe 'Get all repositories of a developer' do
-    it '(HAPPY) should find all repositories with valid group ID' do
-      get "api/v0.1/dev/#{Developer.first.username}/repos"
+  describe 'Get a repository of a developer' do
+    it '(HAPPY) should find a repository from an owner and a repository name' do
```
Comment (rjollet) [inline spec/repo_spec.rb]: this test is good but can you do it with a repo that we do not have in the database

File: spec/repo_spec.rb [hunk 2]
```diff
@@ -14,20 +14,20 @@
...
+      get "api/v0.1/dev/#{Repository.first.full_name}"
       last_response.status.must_equal 200
       last_response.content_type.must_equal 'application/json'
-      repositories = JSON.parse(last_response.body)
-      repositories['repositories'].count.must_be :>=, 5
+      repository = JSON.parse(last_response.body)
```
Comment (rjollet) [inline spec/repo_spec.rb]: can you check if what it return is Repository.first ?

File: services/load_repository_from_github.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# Loads data from Facebook group to database
+class LoadRepositoryFromGithub
+  extend Dry::Monads::Either::Mixin
+  extend Dry::Container::Mixin
+
+  register :check_if_repository_exist, lambda { |repository|
+    github_repo = Github::Repository.find(full_name: repository)
```
Comment (rjollet) [inline services/load_repository_from_github.rb]: GitHub:: Repository.find(owner:, repo:)

**Pull request merged**